src/Datepicker/Calendar/Calendar.js

```diff
@@ -78,8 +78,7 @@ const Calendar = ({
         yearRef={yearRef}
       />
       <table
-        aria-activedescendant={`${selectedDate.month()}-${selectedDate.date()}`}
-        aria-labelledby="datepicker-month-date"
+        aria-activedescendant={`${name}-${selectedDate.month()}-${selectedDate.date()}`}
```

Comment:
Is this causing any axe errors if there is no selected date?

Reply:
by default selected date is today...